KLR/Util/Gzip/Main.lean

```diff
@@ -0,0 +1,71 @@
+import Cli
+import Gzip
+
+open Cli
+
+private def bytesToHex (hash : ByteArray) : String := Id.run do
```

Comment:
Hoping to migrate these bit-twiddling functions to Util at some point soon. Not sure how to do it with multiple lakefiles yet. Probably will need another Util dir with its own lakefile we can `require`.